src/mui/list/List.js

```diff
@@ -17,6 +17,10 @@ import translate from '../../i18n/translate';
 
 const filterFormName = 'filterForm';
 
+const styles = {
+    noResults: { padding: 20, display: 'flex', alignItems: 'flex-end' },
```

Comment:
can we avoid `flex` on this? Normally, all flex styles should be prefixed, and it's cumbersome.